src/yuzu/bootmanager.cpp

```diff
@@ -564,6 +564,15 @@ std::unique_ptr<Core::Frontend::GraphicsContext> GRenderWindow::CreateSharedCont
 bool GRenderWindow::InitRenderTarget() {
     ReleaseRenderTarget();
 
+    {
+        // Create a dummy render widget so that Qt
+        // places the render window at the correct position.
+        auto dummy_widget = new RenderWidget(this);
+        dummy_widget->windowHandle()->create();
+        dummy_widget->deleteLater();
+        dummy_widget = nullptr;
```

Comment:
The fix for this issue can be simplified to the following one-liner.
```suggestion
        const RenderWidget dummy_widget(this);
```